code/weapon/beam.cpp

```diff
@@ -3200,7 +3195,7 @@ void beam_handle_collisions(beam *b)
 			}
 			// <-- KOMET_EXT
 		} else {
-			if(draw_effects && do_damage && !physics_paused){
+			if(draw_effects && !physics_paused){
```

Comment:
This (and other similar changes below) means that impact effects will now be drawn every frame instead of every `BEAM_DAMAGE_TIME` milliseconds. That might be a performance issue and a behavioral change compared to earlier engine versions.